idm/external-idp.yml

```diff
@@ -17,7 +17,7 @@ services:
       PROXY_OIDC_REWRITE_WELLKNOWN: "true"
       WEB_OIDC_CLIENT_ID: ${OC_OIDC_CLIENT_ID:-web}
       PROXY_ROLE_ASSIGNMENT_DRIVER: "oidc"
-      OC_OIDC_ISSUER: https://${KEYCLOAK_DOMAIN:-keycloak.opencloud.test}/realms/openCloud
+      OC_OIDC_ISSUER: ${IDP_ISSUER_URL:-https://keycloak.opencloud.test/realms/openCloud}
```

Comment:
nice! in the helm chart we also use a singel property for this:
https://github.com/opencloud-eu/helm/pull/84/files#diff-66eebf9d371d0ba936f72ba5fe5fa53fe7573e44e06b76fcc82b4bd898d13256L210

maybe `OIDC_ISSUER` is a better env var name?